uniffi_bindgen/src/interface/object.rs

```diff
@@ -290,6 +295,9 @@ impl Constructor {
         self.ffi_func.name = format!("{ci_prefix}_{obj_name}_{}", self.name);
         self.ffi_func.arguments = self.arguments.iter().map(Into::into).collect();
         self.ffi_func.return_type = Some(FfiType::RustArcPtr(obj_name.to_string()));
+
+        // this is a bit of a dirty place to put this, but there isn't another "general" pass
```

Comment:
Can we expand why this is not an ideal place? Inline comment is great, but if we have a more actionable way of making it more ideal then a bug filed with more context would be phenomenal!

Reply:
[#1469 adds an `object_name` field to `Constructor`](https://github.com/mozilla/uniffi-rs/pull/1469/files#diff-db37fade9d78e8f842a50e60a8be13b5eaf0749ee6d0c448179549e5dc6695dc), which would avoid this.  Does that make sense for the long-term plan?